recipes/default.rb

```diff
@@ -0,0 +1,36 @@
+include_recipe 'apt'
+
+type = case node['platform_family']
+       when 'fedora', 'rhel'
+         'rpm'
+       else
+         'deb'
+       end
+
+packagecloud_repo 'StackStorm/stable' do
+  type type
+end
+
+package 'st2mistral'
+
+# ensure directories are managed
+[
+  node['openstack-mistral']['etc_dir']
+].each do |path|
+  directory "creating mistral directory #{path}" do
+    path path
+    owner 'root'
+    group 'root'
+    mode '0755'
+    action :create
+  end
+end
+
+template '/etc/mistral/mistral.conf' do
+  source 'mistral.conf.erb'
+  variables(
+    options: node['openstack-mistral']['config']
+  )
+  mode '0644'
+  action :create
+end
```

Comment:
Shouldn't we include `notifies` somewhere in this section to trigger `st2mistral` service restart, once config was changed?